src/overlay/test/SurveyManagerTests.cpp

```diff
@@ -438,8 +434,21 @@ TEST_CASE("survey request process order", "[overlay][survey][topology]")
             false);
     };
 
+    // Get survey into reporting mode
+    auto startSurveyReportingFrom = [&](PublicKey const& node) {
```

Comment:
Let's avoid duplication of logic with line 200, and extract it into a static function.

Reply:
Done